src/lib/sedlexing.ml

```diff
@@ -436,23 +518,24 @@ module Utf16 = struct
         Some (Uchar.of_int (0x10000 + upper10 + lower10)))
       else raise MalFormed
 
+    let get_bo bo c1 c2 =
+      match !bo with
+        | Some o -> o
+        | None ->
+            let o =
+              match (Char.code c1, Char.code c2) with
+                | 0xff, 0xfe -> Little_endian
+                | _ -> Big_endian
```

Comment:
Minor comment: Even though the unicode standard says that this is correct behaviour, wikipedia says it's not really true in practice, people use little ending without a bom

Reply:
Not sure what to do with that, maybe create a new issue for this ?

Reply:
yeah, it's the same as the existing behaviour -- my comment was just for future reference